### ISSUE TITLE
NullPointerException in vertx 4.3.2

### ISSUE THREAD

**Issue by glassfox** [eclipse-vertx/vertx-junit5#119]
vertx 4.3.2

Exception:
```
java.lang.NullPointerException: Cannot invoke "io.vertx.core.spi.logging.LogDelegateFactory.createDelegate(String)" because "io.vertx.core.logging.LoggerFactory.delegateFactory" is null
    at io.vertx.core.logging.LoggerFactory.getLogger(LoggerFactory.java:93)
    at io.vertx.core.logging.LoggerFactory.getLogger(LoggerFactory.java:82)
    at io.vertx.core.impl.logging.LoggerFactory.getLogger(LoggerFactory.java:26)
    at io.vertx.core.impl.VertxImpl.<clinit>(VertxImpl.java:90)
```

Reproducer:
```java
package io.vertx.ext.mesh;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;

import io.vertx.core.Vertx;
import io.vertx.junit5.VertxExtension;

@ExtendWith(VertxExtension.class)
class MeshTest {

    @Test
    void test(Vertx vertx) {
    }
}
```

**Comment by jponge** [comment]
I've tested locally on `4.3.3-SNAPSHOT` and I cannot reproduce the error.

Would you have a complete minimal reproducer project?

**Comment by glassfox** [comment]
I will check later

**Comment by glassfox** [comment]
After short investigation, has been found that issue not in JUnit but in core. 
Fixed in vertx 4.3.4

https://github.com/eclipse-vertx/vert.x/pull/4459
